package.json

```diff
@@ -12,6 +12,7 @@
     "test": "mocha --harmony tests/**",
     "test-frontend": "mocha --harmony examples/multiple-fragments-with-custom-amd/test.js",
     "coverage": "node --harmony node_modules/.bin/istanbul cover _mocha -- tests/**",
+    "codecov": "cat coverage/*/lcov.info | codecov",
```

Comment:
But do we want to club these? If these are isolated we can run the coverage even on local. Thoughts?